react/external_data.rst

```diff
@@ -0,0 +1,121 @@
+.. _external_data-label:
+
+===================
+Using External Data
+===================
+
+Creating A Simple Backend
+=========================
+
+In order to persist our data we will create a backend to fetch our initial data.
+We will use :file:`express` to create a simple server. To install type:
+
+.. code-block:: console
+
+    $ yarn add express
+
+Now we'll create a simple server in the file :file:`server.js`
+
+::
```

Comment:
```
    .. code-block::
        :language: js
```

Reply:
Please use `we will` not everyone is that fluent in englisch as you are, `we will` is easier to understand :)